### ISSUE TITLE
0.6.0 Fails to Create Accessories

### ISSUE THREAD

**Issue by rcoletti116** [dotfortun3-code/homebridge-infinitude-v2#8]
Latest version 0.6.0 fails to create the thermostat accessory; sends homebridge instance into loop.

 Here are the logs.

```
Creating` new thermostat in zone: Zone 1 ZONE 1 Thermostat

/usr/local/lib/node_modules/homebridge-infinitude-v2/src/InfinitudeInstance.js:80
    this.configureThermostat(zoneAccessory);
         ^
TypeError: this.configureThermostat is not a function
    at InfinitudeInstance.createZoneThermostat (/usr/local/lib/node_modules/homebridge-infinitude-v2/src/InfinitudeInstance.js:80:10)
    at InfinitudeInstance.<anonymous> (/usr/local/lib/node_modules/homebridge-infinitude-v2/src/InfinitudeInstance.js:57:47)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
```

For this, I think InfinitudeInstance.js line 80 should read `this.configureZoneThermostat.`

Once you address that though there is another problem though.

```
[25/08/2022, 20:57:19] AssertionError [ERR_ASSERTION]: Accessories must be created with a valid UUID.
    at new Accessory (/usr/local/lib/node_modules/homebridge/node_modules/hap-nodejs/src/lib/Accessory.ts:422:11)
    at new PlatformAccessory (/usr/local/lib/node_modules/homebridge/src/platformAccessory.ts:70:9)
    at InfinitudeInstance.createTemperatureSensor (/usr/local/lib/node_modules/homebridge-infinitude-v2/src/InfinitudeInstance.js:85:29)
    at InfinitudeInstance.<anonymous> (/usr/local/lib/node_modules/homebridge-infinitude-v2/src/InfinitudeInstance.js:63:55)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
```

**Comment by rcoletti116** [comment]
@dotfortun3-code making sure you saw this. I can't get the current version of the plugin to work at all and rolled back. I was working on a PR, but each item I've addressed uncovers something new. I think the way the 0.6.0 is generating IDs is broken, but I haven't found the culprit yet.

**Comment by dotfortun3-code** [comment]
Just saw this. I will take a look and make the changes. Not sure what happened.

**Comment by dotfortun3-code** [comment]
Should be fixed in v0.6.1.
